src/test/DripsHub.t.sol

```diff
@@ -164,22 +116,6 @@ contract DripsHubTest is DripsHubUserUtils {
         collectAll(receiver2, 10);
     }
 
-    function testCollectAllSplitsFundsFromSplits() public {
```

Comment:
The `collectAll` is a DripsHub-specific function, is it the right choice to move these tests to `Splits.t.sol`? They don't seem to test there anything more than simple splitting.